api/src/pcapi/core/bookings/api.py

```diff
@@ -157,10 +157,16 @@ def get_individual_bookings(user: User) -> list[Booking]:
                 Venue.longitude,
                 Venue.publicName,
                 Venue.timezone,
+                Venue.bannerUrl,
+                Venue.isOpenToPublic,
+                Venue.venueTypeCode,
             )
             .joinedload(Venue.offererAddress)
             .joinedload(OffererAddress.address)
         )
+        .options(
+            joinedload(Booking.stock).joinedload(Stock.offer).joinedload(Offer.venue).joinedload(Venue.googlePlacesInfo)
+        )
```

Comment:
[question] 
Je comprends pas trop pourquoi on ajoute pas juste 
```suggestion
    .joinedload(Venue.googlePlacesInfo)
```

Reply:
Je vois pas le lien plus court vers la `Venue`, pour moi on est obligé de suivre le chemin qu'a mis Bruno

Reply:
en effet, l'orm demande le lien complet sinon
`Mapped attribute "Venue.googlePlacesInfo" does not apply to any of the root entities in this query, e.g. mapped class Booking->booking. Please specify the full path from one of the root entities to the target attribute.`

Reply:
Je savais pas 😮